server/sonar-server/src/main/java/org/sonar/server/rule/RuleService.java

```diff
@@ -77,7 +75,9 @@ public RuleQuery newRuleQuery() {
   }
 
   public Result<Rule> search(RuleQuery query, QueryContext options) {
-    return index.search(query, options);
+    // TODO replace QueryContext by SearchOptions
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)